src/utils/DashboardHelperFunctions.js

```diff
@@ -68,23 +69,29 @@ export const applyEdits = (oktaAxios, formValues, incident) => {
   return putRequest;
 };
 
-// AddIncident.js
+// MapQuest API to get Latitude/Longitude used in clusters (Clusters.js)
 export const getLatAndLong = formValues => {
   const getRequest = new Promise((resolve, reject) => {
     const { city, state } = formValues;
+
+    if (!city || !state) {
+      throw new Error('Missing city or state');
```

Comment:
This extra error handling for an edge case where the user makes an error and forgets to fill out both city & state was a nice touch!